src/utils.py

```diff
@@ -120,6 +120,7 @@ def get_webdriver(proxy: dict = None) -> WebDriver:
     options = uc.ChromeOptions()
     options.add_argument('--no-sandbox')
     options.add_argument('--window-size=1920,1080')
+    options.add_argument('--auto-open-devtools-for-tabs')
```

Comment:
Can you confirm it is working without this line? I'm finding that only the change to flaresolverr_service.py is necessary.

Reply:
My understanding is the purpose this line serves is opening the dev tools, which is needed as part of this fix

Reply:
Having tested on both Windows and Docker, it doesn't seem to be though.

Reply:
You are right, it seems to work without this option. It's weird because I couldn't get it to work without it earlier today.
I'll push a commit to remove it from the list.